### DISCUSSION TITLE
Messages

### DISCUSSION THREAD

**Discussion by robertjmccarthy** [ministryofjustice/moj-frontend#705] [RELEASED: Components, pages and patterns]
The [messages component](https://design-patterns.service.justice.gov.uk/components/messages/) is live in the MoJ Design System.

Use this discussion to suggest changes and improvements to this component.

**Comment by Whersi** [comment]
## Ticket number

https://github.com/ministryofjustice/moj-frontend/issues/63

The details of the ticket is as follows:

To support realtime messaging, update the mojMessages component to include aria live regions.
